addon/mixins/ajax-request.ts

```diff
@@ -434,16 +434,16 @@ export default Mixin.create({
    */
   _getFullHeadersHash(headers?: Headers): Headers {
     const classHeaders = get(this, 'headers');
-    const _headers = merge({}, classHeaders);
-    return merge(_headers, headers);
+    const _headers = assign({}, classHeaders);
+    return assign(_headers, headers);
```

Comment:
Can you refactor this to a single assign call?

    const _headers = assign({}, classHeaders, headers);

Reply:
Done